src/components/Navigation/Navigation.tsx

```diff
@@ -13,12 +16,23 @@ export const Navigation = () => {
       <Link className="navbar-item" to="/">
         Home
       </Link>
-      <Link className="navbar-item" to="/signin">
-        Sign in
-      </Link>
-      <Link className="navbar-item" to="/album">
-        Album
-      </Link>
+      {authContext.userId === null ? (
```

Comment:
Niiice... I know we wanted to have this displayed differently, but nonetheless I like how this works!